tests/test_base_template.py

```diff
@@ -58,7 +58,7 @@ def test_matches_keys_to_placeholder_names():
     assert template.additional_data == {'Town'}
 
     template.values = None
-    assert template.missing_data == ['name']
+    assert template.missing_data == {'name'}
```

Comment:
above this, on line 49, the test is defined as d`ef test_matches_keys_to_placeholder_names():` and the template as `template = Template({"content": "hello ((name))"})`. Does content cover a personalisation in the subject field? or just in the body? Does it matter?